CHANGELOG-nightly.md

```diff
@@ -8,6 +8,9 @@ will be backported to the master branch at every major release.
 When backporting changes to master, we should keep only the entries that correspond to user-
 facing changes.
 -->
+- 💥[Feature] Add an extensible `local/dev/k8s do ...` command to trigger custom job commands. These commands are used to run a series of bash scripts in designated containers. Any plugin can add custom jobs thanks to the `CLI_DO_COMMANDS` filter. This causes the following breaking changes:
+    - The "createuser", "settheme", "importdemocourse" commands were all migrated to this new interface. For instance,  `tutor local createuser` was replaced by `tutor local do createuser`.
+    - The `init` command loses its `-m/--mount` option  (which was ignored by `quickstart`).
```

Comment:
I'm not sure I understand this. Launch/quickstart supports mounting, so why wouldn't init?